hazelcast/src/main/java/com/hazelcast/map/IMap.java

```diff
@@ -1720,15 +1735,18 @@ V putIfAbsent(@Nonnull K key, @Nonnull V value,
      * max idle time value without returning the old value (which is more
      * efficient than {@code put()}).
      * <p>
-     * The entry will expire and get evicted after the TTL. If the TTL is 0,
-     * then the entry lives forever. If the TTL is negative, then the TTL
-     * from the map configuration will be used (default: forever).
-     * <p>
-     * The entry will expire and get evicted after the Max Idle time. If the
-     * MaxIdle is 0, then the entry lives forever. If the MaxIdle is negative,
-     * then the MaxIdle from the map configuration will be used
-     * (default: forever). The time precision is limited by 1 second. The MaxIdle
-     * that less than 1 second can lead to unexpected behaviour.
+     * The entry will expire and get evicted after the TTL. It limits the
```

Comment:
why don't you put these descriptions to the parameters itself?

Reply:
They already have short descriptions. As for this particular descriptions, they were already present in the javadoc and that approach seems to be consistent across `IMap` interface.

Reply:
Please let me know if there is a reason to move the clarifications into parameter descriptions. Then I'll submit a separate PR to change that.